components/brave_wallet/browser/brave_wallet_service.cc

```diff
@@ -36,6 +37,11 @@ BraveWalletService::BraveWalletService(
       std::make_unique<brave_wallet::AssetRatioController>(url_loader_factory);
   swap_controller_ =
       std::make_unique<brave_wallet::SwapController>(url_loader_factory);
+  block_tracker_ =
+      std::make_unique<brave_wallet::EthBlockTracker>(rpc_controller_.get());
+  // TODO(darkdh): Stop the tracker to save bandwidth when users are not using
+  // wallet actively
+  block_tracker_->Start(base::TimeDelta::FromSeconds(15));
```

Comment:
Maybe we shouldn't start it yet until we need to use it for something? 
E.g. I think maybe only when a WebUI is open

Reply:
With our current wallet we only start the block tracker when the UI is open. And close it even if the tab is not active. And we lazy load the extension, and traffic is already very high even with that restricted way.

Reply:
I will remove it for now since we are not really using it

Reply:
removed